Kite SDK/PSPrintSDK/OLProduct.h

```diff
@@ -27,5 +27,7 @@
 -(void)setCoverImageToImageView:(UIImageView *)imageView;
 -(void)setProductPhotography:(NSUInteger)i toImageView:(UIImageView *)imageView;
 +(OLProduct *) productWithTemplateId:(NSString *)templateId;
+- (NSDecimalNumber*) decimalNumberUnitCost;
++ (NSString*) unitCostWithCost:(NSDecimalNumber*)cost;
```

Comment:
Again I'd probably make this a private internal method that we use elsewhere by including the appropriate private interface, or do you really think it should be public?